catalog/static/oauth-callback.html

```diff
@@ -1,8 +1,11 @@
-<!doctype html>
+<!DOCTYPE html>
 <html lang="en">
   <body>
     <script>
-      window.opener.postMessage({type: "callback", fragment: window.location.hash}, window.location.origin);
+      window.opener.postMessage(
+        { type: 'callback', fragment: window.location.hash },
+        window.location.origin,
+      )
```

Comment:
FYI, this file is served as is, without being compiled/processed, so this change could theoretically break things: the trailing comma was not supported before 2017: https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Trailing_commas#browser_compatibility

(Maybe it should be compiled? I'm not sure how to set it up properly.)

Reply:
Thanks for this find!
I'll redefine prettier config for static directory, so it will use ES5 rules for commas